privacy-contracts/src/main/solidity/DefaultOnChainPrivacyGroupManagementContract.sol

```diff
@@ -4,11 +4,11 @@ import "./OnChainPrivacyGroupManagementInterface.sol";
 contract DefaultOnChainPrivacyGroupManagementContract is OnChainPrivacyGroupManagementInterface {
 
     bool private _canExecute;
-    int private _version;
+    bytes32 private _version;
     bytes32[] private distributionList;
     mapping(bytes32 => uint256) private distributionIndexOf;
 
-    function getVersion() external view returns (int) {
+    function getVersion() external view returns (bytes32) {
```

Comment:
Have you tested the impact of this change in web3j and web3js-eea? If they aren't compatible, we need at least to have GIs to update the libraries after we change this.

Reply:
There should not be any public APIs for the version number in web3j(s), as this is entirely handled inside Besu. But we should check that!